corehq/db/config.yml

```diff
@@ -0,0 +1,20 @@
+# HQ database configuration for multiple database. This gets mapped to Django's DATABASES dict
```

Comment:
Good point about this being a pain locally. I'm not convinced about the SO solution you linked to. I think that works for basic key value settings but not for something like this which is more complex and going to look very different on different environments.

What do you think about putting it in a separate .py file that gets imported by settings (basically splitting settings.py up into multiple files)?

To keep this easy for local usage perhaps we need to add `USE_PARTITIONED_DB=False` to `settings.py` that short circuits the multi-db config and just uses the `default` database.